### PR TITLE
[SPARK-37635][SQL] SHOW TBLPROPERTIES should print the fully qualified table name

### PR DESCRIPTION
**Pull request by cloud-fan**
### What changes were proposed in this pull request?

This fixes a small issue in the v2 SHOW TBLPROPERTIES command, where we should print the fully qualified table name, instead of a implementation-provided table "description".

### Why are the changes needed?

bug fix

### Does this PR introduce _any_ user-facing change?

SHOW TBLPROPERTIES now prints fully qualified table name.

### How was this patch tested?

existing tests

### CODE REVIEW

**Comment by cloud-fan** [comment]
cc @imback82 @huaxingao

File: sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/DataSourceV2Strategy.scala [hunk 1]
```diff
@@ -353,7 +353,8 @@ class DataSourceV2Strategy(session: SparkSession) extends Strategy with Predicat
       SetCatalogAndNamespaceExec(catalogManager, Some(catalog.name()), namespace) :: Nil
 
     case ShowTableProperties(rt: ResolvedTable, propertyKey, output) =>
-      ShowTablePropertiesExec(output, rt.table, propertyKey) :: Nil
+      val tableName = (rt.catalog.name() +: rt.identifier.asMultipartIdentifier).quoted
```
Comment (imback82) [inline sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/DataSourceV2Strategy.scala:356]: Can you use `rt.name` instead?

**Review by imback82 — APPROVED** [review]
LGTM (one comment), thanks @cloud-fan!

**Review by yaooqinn — APPROVED** [review]
LGTM

**Review by huaxingao — APPROVED** [review]

**Review by sarutak — APPROVED** [review]

**Comment by sarutak** [comment]
Merged to `master` as GA passed. Thank you @cloud-fan !

**Pull request closed without merging**